src/njoy21/input/VIEWR/Card1/Nps.hpp

```diff
@@ -0,0 +1,28 @@
+struct Nps { 
```

Comment:
The verify function for `argument::common::nout` can be called from the verify function if needed. However, if `Nps` inherits from `argument::common::nout`, then the verify function needs not be reimplemented—unless a change is to be made.